report/report.md

```
@@ -0,0 +1,602 @@
+    Confirmation --> |Browse More| Browse
+    
+    SeatSelection --> |Timer Expires| AutoUnlock[Auto Unlock Seats]
+    SeatSelection --> |Navigate Away| ManualUnlock[Manual Unlock Seats]
+    
+    style Start fill:#e1f5fe
+    style Confirmation fill:#c8e6c9
+    style ErrorMsg fill:#ffcdd2
+```
+-----
+*1. Landing page showing featured movies*
+![](images/screenshot_01.png)
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)